tests/i18n/patterns/tests.py

```diff
@@ -92,6 +93,18 @@ def test_prefixed_i18n_disabled(self):
             self.assertEqual(reverse('prefixed'), '/prefixed/')
 
 
+class RequestURLConfTests(URLTestCaseBase):
```

Comment:
I agree that the tests would benefit from a refactor. I'll just use `SimpleTestCase` w/ `override_settings` for now.